IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,102 @@
+import UIKit
+
+var currencyFormatter: NumberFormatter = {
+    let formatter = NumberFormatter()
+    formatter.numberStyle = .currencyAccounting
+    return formatter
+}()
+
+struct Flavor {
+    let name: String
+    let rating: Double
+}
+
+enum Size: Double {
+    case small = 3.99
+    case medium = 4.99
+    case large = 5.99
+}
+
+struct Cone {
+    var flavor: Flavor
+    var toppings: String
+    var size: Size
+    
+    func eat(){
+        print("Mmmm! I love \(flavor.name)!!")
+    }
+}
+
+class IceCreamShop {
+    var flavor: [Flavor]
+    var size: [Size]
+    var toppings: [String]
+    var totalSales: Double = 0.00
+    
+    init(flavor: [Flavor],size: [Size],toppings: [String], totalSales: Double) {
+        self.flavor = flavor
+        self.size = size
+        self.toppings = toppings
+        self.totalSales = totalSales
+    }
+    
+    func listTopFLavors() {
+        
+        for flavor in flavor {
+            if flavor.rating >= 4.0 {
+                print("Our top flavors are: \(flavor.name)")
+            }
+        }
+    }
```

Comment:
I would suggest modifying this function to print out the top flavors all in one print statement. Right now it prints the following:

> Our top flavors are: Chocolate
Our top flavors are: Vanilla
Our top flavors are: pistachio

It would read better if it was:

> Our top flavors are chocolate, vanilla, pistachio

How would you go about rewriting your code to accomplish this output?